crates/rspack_hash/src/lib.rs

```diff
@@ -78,7 +84,8 @@ impl RspackHash {
 
   pub fn digest(self, digest: &HashDigest) -> RspackHashDigest {
     let inner = match self {
-      RspackHash::Xxhash64(hasher) => hasher.finish(),
+      RspackHash::Xxhash64(hasher) => hasher.finish().to_le_bytes().to_vec(),
+      RspackHash::MD4(hash) => hash.finalize().to_vec(),
```

Comment:
Aiming to support a length of `20` hash, only return `u64` is not enough, because u64 can only hold `8bytes` data, so I use a `Vec` here.